### PR TITLE
Regenerate tests

### PR DESCRIPTION
**Pull request by jbrette**
- Exclude kfdef
- Account for directory name with dot
- Rerun generation

**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**

**Checklist:**
- [ ] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate-changed-only`
    3. `make test`

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/831)

### CODE REVIEW

File: hack/gen-test-target.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ kebab-case-2-PascalCase() {
   local a=$1 b='' array
   IFS='-' read -r -a array <<< "$a"
   for element in "${array[@]}"; do
-    part="${element}"
+    part="${element//./_}"
```
Comment (jlewi) [inline hack/gen-test-target.sh]: What is this change for?

Comment (jbrette) [reply hack/gen-test-target.sh]: Simply deals with directory names which are like these : components/v1.0.0/....
The generated go code should not contains `.` in the function names. The code change replaces it with _

**Comment by jlewi** [comment]
/test all

Looks like the 2nd apply test failed; I think @gabrielwen is working on fixing that; you might want to talk to him if its not a flake.

/lgtm
/approve

**Comment by jlewi** [comment]
See 
https://github.com/kubeflow/manifests/pull/821#issuecomment-582101210

Regarding the test failure.

**Pull request merged**